tools/pyclient/src/molgenis_emx2_pyclient/data_classes.py

```diff
@@ -0,0 +1,288 @@
+"""
+Classes for the data types Schema, Table and Column.
+"""
+from dataclasses import dataclass
+from typing import Literal
+
+from tools.pyclient.src.molgenis_emx2_pyclient.exceptions import NoSuchColumnException, NoSuchTableException
+
+
+@dataclass(eq=True, kw_only=True)
+class Column:
+    """Class for representing a column in the database."""
+    id: str
+    name: str
+    table: str
+    label: str
+    description: str
+    labels: dict
+    description: dict
+    position: int
+    columnType: str
+    inherited: bool
+    key: int
+    required: str
+    defaultValue: str
+    refSchemaId: str
+    refSchemaName: str
+    refTableName: str
+    refTableId: str
+    refLinkName: str
+    refLinkId: str
+    refBackName: str
+    refLabel: str
+    refLabelDefault: str
```

Comment:
this is also an autogenerated field, when refLabel is not provided it will be generated based on key=1

Reply:
In order to improve maintainability, I have now removed the type hints for most of the potential attributes for the classes.

So as long as Tables have ids, names, and columns and columns have ids and names, this representation will be compatible with the database/GraphQL responses